source/garbageHandler.py

```diff
@@ -49,9 +51,9 @@ def _collectionCallback(action, info):
 	elif action == "stop":
 		_collectionThreadID = 0
 		if _reportCountDuringCollection > 0:
-			log.error(f"Found at least {_reportCountDuringCollection} unreachable objects in run")
+			log.debugWarning(f"Found at least {_reportCountDuringCollection} unreachable objects in run")
 	else:
-		log.error(f"Unknown action: {action}")

```

Comment:
Shouldn't this remain logged at error level, so that if there are any additions to the actions supported by Python's `gc` interface, we are warned in advance?